package/reboot-guard/package

```diff
@@ -5,7 +5,7 @@
 pkgname=reboot-guard
 pkgdesc="Block systemd-initiated poweroff/reboot/halt until configurable condition checks pass"
 url=https://github.com/stephanritscher/reboot-guard
-pkgver=1.0.1~1
+pkgver=1.0.1-1
```

Comment:
thanks. i need to get a better eye for this

Reply:
Or we need to work on #8 :)